app/cafe.py

```diff
@@ -0,0 +1,23 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+
+class Cafe:
+    def __init__(self, name):
+        self.name = name
+
+    def visit_cafe(self, visitor: dict):
+        today = date.today()
+
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
```

Comment:
you should pass descriptive messages when you raise the exception in the visit_cafe method